src/main/content/_posts/microprofile-openapi-intro.adoc

```diff
@@ -0,0 +1,61 @@
+
+The consortium governance model of this new specification meant that new tools vendors started to emerge with libraries and SDKs.  Java developers had a choice from many different annotations or libraries to include in their application – but none that was officially supported by any of the major Java application server vendors, which affected portability and consistency.  There was an increasing need for an official Java programming model that was supported by different vendors and integrated with other modern libraries.  
+
+## Enter MicroProfile OpenAPI
+
+Similarly to how the Swagger specification was used as the base for a community-driven specification, the Java libraries for Swagger were used as the base for the http://microprofile.io/)[MicroProfile community] programming model for OpenAPI v3: https://github.com/eclipse/microprofile-open-api[MicroProfile OpenAPI].  This specification outlined an official set of annotations and interfaces that allow developers to successfully generate OpenAPI v3 documentation for their Java RESTful APIs.
+
+The following diagram provides an overview of the MicroProfile OpenAPI support in Open Liberty, via the `mpOpenAPI-1.0` feature.
+
+image::img/mp_ol_openapi.png[MP OpenAPI Architecture]
+
+One of the key aspects of the MicroProfile OpenAPI specification is the requirement to produce a valid OpenAPI document from pure JAX-RS 2.0 applications. This means that Open Liberty processes all the relevant JAX-RS annotations (such as `@Path` and `@Consumes`) as well as Java objects (POJOs) used as input or output to JAX-RS operations. This is a good place to start for application developers that are new to OpenAPI: just deploy your existing JAX-RS application into Open Liberty and check out the output from /openapi!
+
+The application developer then has a few choices to provide input for the generation of the resulting OpenAPI document:
+
+1.	Augment those JAX-RS annotations with the OpenAPI https://github.com/eclipse/microprofile-open-api/blob/master/spec/src/main/asciidoc/microprofile-openapi-spec.adoc#annotations[Annotations]. Using annotations means developers don’t have to re-write the portions of the OpenAPI document that are already covered by the JAX-RS framework (e.g. the HTTP method of an operation).  This is the most common path, so definitely check out some of the https://github.com/eclipse/microprofile-open-api/blob/master/spec/src/main/asciidoc/microprofile-openapi-spec.adoc#412-detailed-usage-of-key-annotations[examples] to get familiar with some of these annotations and its generated output.
```

Comment:
I think the end of the last sentence of this paragraph should be "... and **their** generated output" since we're talking about multiple examples.